context/app/static/js/components/detailPage/StatusIcon/ColoredStatusIcon/style.ts

```diff
@@ -1,20 +1,7 @@
 import React from 'react';
 import { styled } from '@mui/material/styles';
-import { InfoRounded, ErrorRounded, WarningRounded, CheckCircleRounded } from '@mui/icons-material';
 import { SvgIconProps } from '@mui/material/SvgIcon';
-
-export interface ColoredStatusIconProps extends SvgIconProps {
-  $iconStatus: 'info' | 'success' | 'warning' | 'error';
-}
-
-export const iconSymbolStatusMap: {
-  [key in ColoredStatusIconProps['$iconStatus']]: React.ComponentType<SvgIconProps>;
-} = {
-  info: InfoRounded,
-  success: CheckCircleRounded,
-  warning: WarningRounded,
-  error: ErrorRounded,
-};
+import { ColoredStatusIconProps } from 'js/shared-styles/icons/SeverityIcon';
```

Comment:
We should be able to get rid of this file, use `SeverityIcon` directly in `StatusIcon` and style it there using the `sx` prop to apply its unique styling there.